x-pack/plugins/ml/public/application/explorer/explorer_charts/components/explorer_chart_label/influencer_filters/influencer_filter.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { FC } from 'react';
+import { EuiButtonIcon, EuiToolTip } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+
+type Operation = '+' | '-';
+interface InfluencerFilterProps {
+  onFilter: (params: {
+    influencerFieldName: string;
+    influencerFieldValue: string;
+    operation: Operation;
+  }) => void;
+  influencerFieldName: string;
+  influencerFieldValue: string;
+}
+export const InfluencerFilter: FC<InfluencerFilterProps> = ({
+  onFilter,
+  influencerFieldName,
+  influencerFieldValue,
+}) => {
+  return (
+    <React.Fragment>
```

Comment:
It doesn't seem like you need a fragment

Reply:
We have 2 components here so I'll keep the fragment for now.